doc/cookbook/source/index.rst

```diff
@@ -105,3 +105,13 @@ Neural Networks
    :caption: Neural Netwroks
 
    examples/neural_nets/**
+
+Feature Transformations
```

Comment:
Glad to change it, but are you sure about it? I haven't come across the term "Feature converters" in the machine learning literature.